src/include/schema.h

```diff
@@ -160,7 +161,7 @@ struct __wt_import_list {
     WT_WITH_LOCK_WAIT(session, &S2C(session)->checkpoint_lock, WT_SESSION_LOCKED_CHECKPOINT, op)
 #define WT_WITH_CHECKPOINT_LOCK_NOWAIT(session, ret, op) \
     WT_WITH_LOCK_NOWAIT(                                 \
-      session, ret, &S2C(session)->checkpoint_lock, WT_SESSION_LOCKED_CHECKPOINT, op)
+      session, ret, ret, &S2C(session)->checkpoint_lock, WT_SESSION_LOCKED_CHECKPOINT, op)
```

Comment:
@jiechenbo, what do you think of this "ret, ret". It is going to become "ret, __lock_ret" when if we create something similar to the schema lock, such as "WT_CONFLICT_CKPT_LOCK"?

Reply:
In this case, i do find "ret, ret"  to be slightly weird, but I am happy to go ahead with it. I believe if we want to create a sub-level error code such as "WT_CONFLICT_CKPT_LOCK", it will become "ret, __lock_ret".

On another related note, WT_WITH_CHECKPOINT_LOCK_NOWAIT is something we missed from the original tech design. We will be adding a new sub-level error code addressing this. So I am happy for now to leave it as is.